kokoro/build_binaries_rust.sh

```diff
@@ -42,8 +42,8 @@ readonly generated_binaries=(
     ./enclave_apps/target/x86_64-unknown-none/release/oak_orchestrator
 )
 readonly binary_names=(
-    oak_restricted_kernel_simple_io_wrapper_bin
-    oak_restricted_kernel_simple_io_init_rd_wrapper_bin
+    oak_restricted_kernel_simple_io_bzimage
+    oak_restricted_kernel_simple_io_init_rd_bzimage
```

Comment:
essentially these binary names must match a subject in the provenance. 

This is already the case, but not entirely obvious from the code. We should add additional comments for this.